presto-raptor/src/main/java/com/facebook/presto/raptor/systemtables/PreparedStatementBuilder.java

```diff
@@ -98,6 +103,9 @@ private static String getWhereClause(
             Type type = types.get(index);
```

Comment:
on line 99 tuple domain can be none. I think you need a separate condition to verify it